packages/app/src/cli/utilities/developer-platform-client/app-management-client.test.ts

```diff
@@ -187,3 +190,62 @@ describe('versionDeepLink', () => {
     expect(got).toEqual('https://dev.shopify.com/dashboard/1/apps/2/versions/3')
   })
 })
+
+describe('searching for apps', () => {
+  async function appSearchTest({query, queryVariable}: {query?: string; queryVariable: string}) {
+    // Given
+    const orgId = '1'
+    const appName = 'test-app'
+    const apps = [testOrganizationApp({title: appName})]
+    const mockedFetchAppsResponse = {
+      appsConnection: {
+        edges: apps.map((app, index) => ({
+          node: {
+            ...app,
+            key: `key-${index}`,
+            activeRelease: {
+              id: 'gid://shopify/Release/1',
+              version: {
+                name: app.title,
+                appModules: [],
+              },
+            },
+          },
+        })),
+        pageInfo: {
+          hasNextPage: false,
+        },
+      },
+    }
+    vi.mocked(appManagementRequestDoc).mockResolvedValueOnce(mockedFetchAppsResponse)
+
+    // When
+    const client = new AppManagementClient()
+    client.token = () => Promise.resolve('token')
+    const got = await client.appsForOrg(orgId, query)
+
+    // Then
+    expect(vi.mocked(appManagementRequestDoc)).toHaveBeenCalledWith(orgId, ListApps, 'token', {query: queryVariable})
+    expect(got).toEqual({
+      apps: apps.map((app, index) => ({
+        apiKey: `key-${index}`,
+        id: app.id,
+        organizationId: app.organizationId,
+        title: app.title,
+      })),
+      hasMorePages: false,
+    })
+  }
+
+  test('passes in a blank search if none is provided', async () => {
+    await appSearchTest({query: undefined, queryVariable: ''})
```

Comment:
Given that the default behaviour is `term = ''` can we get a test asserting that query stays empty in that case.